services/event/db/schema.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require('mongoose')
+const Joigoose = require('joigoose')(mongoose)
+const Joi = require('joi')
+
+const joiEventSchema = Joi.object({
+  dbType: Joi.string().required(),
+  name: Joi.string().required().meta({ unique: true }),
+  dbURL: Joi.string().required(),
+  regCount: Joi.number().required().meta({ default: 0 }),
```

Comment:
`regCount` can't be a property of an Event, this is unlikely to work structurally in the future

An event can have multiple tables, and the count can be from any one of those tables.

Reply:
done